tests/components/plugwise/test_binary_sensor.py

```diff
@@ -53,7 +54,20 @@ async def test_adam_climate_binary_sensor_change(
     hass: HomeAssistant, mock_smile_adam: MagicMock, init_integration: MockConfigEntry
 ) -> None:
     """Test change of climate related binary_sensor entities."""
-    state = hass.states.get("binary_sensor.adam_plugwise_notification")
+    entity_id = "binary_sensor.adam_plugwise_notification"
+    # Test disabled_by default entry
+    assert hass.states.get(entity_id) is None
+
+    # Enable entry
+    init_integration.add_to_hass(hass)
+    registry = er.async_get(hass)
+    registry.async_update_entity(
+        "binary_sensor.adam_plugwise_notification", disabled_by=None
+    )
+    await hass.config_entries.async_reload(init_integration.entry_id)
+    await hass.async_block_till_done()
```

Comment:
Use the available `entity_registry_enabled_by_default` fixture instead, it removes the need to manually register and enable these entities.

Reply:
This change was also reverted as the binary_sensor is changed back to default-enabled.